scripts/check-size-firmware.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/sh
+
+DTB=$1
+IMAGE_FILE_SIZE=$2
+PARTITION_NODES="/ahb/spi/flash/partitions /spi/flash/partitions"
+FDGET=fdtget
+
+echo "checking $(basename $DTB)"
+for part_node in $PARTITION_NODES; do
+	for partition in $($FDGET -l $DTB $part_node 2>/dev/null); do
+		name=$($FDGET $DTB $part_node/$partition label 2>/dev/null)
+		[ "$name" = "firmware" ] && {
```

Comment:
I'm not sure if hardcoding `firmware` makes sense here, maybe it should be parameter as well?

Reply:
Done.